pkg/controller/garbagecollector/garbagecollector_test.go

```diff
@@ -856,7 +856,17 @@ func TestGarbageCollectorSync(t *testing.T) {
 	stopCh := make(chan struct{})
 	defer close(stopCh)
 	go gc.Run(1, stopCh)
-	go gc.Sync(fakeDiscoveryClient, 10*time.Millisecond, stopCh)
+	// The pseudo-code of GarbageCollector.Sync():
+	// GarbageCollector.Sync():
```

Comment:
nit: add the input parameters, e.g., GarbageCollector.Sync(client, period, stopCh). Otherwise readers won't know what "period" we are referring to.